private-templates-service/client/src/App.tsx

```diff
@@ -111,6 +113,10 @@ class App extends Component<Props, State> {
                 <Route exact path="/">
                   <Dashboard authButtonMethod={this.login} />
                 </Route>
+                <Route exact path="/designer">
+                  <Designer authButtonMethod={this.login} />
```

Comment:
Why does this component need `authButtonMethod`

Reply:
Designer is wrapped by requireAuthentication (HOC). HOC displays the Welcome component if user is not authenticated. Since Welcome component takes authButtonMethod, the HOC along with any components wrapped by the HOC must take authButtonMethod.